scss/style.scss

```diff
@@ -36,89 +33,160 @@ header {
-    margin-left:91px;
-}
-.header-buttons{
-    margin:69px 96px;
-}
-.arrows{
-    position: absolute;
-    width:inherit;
-}
-.header-buttons button{
-    cursor: pointer;
-    font-size: 16px;
-    color:#FAFAFA;
-    width:150px;
-    height: 50px;
-    border:1px solid white;
-    background:none;
-}
-.slider-dots{
-    margin:140px auto;
-    
-    display:flex;
-    justify-content: space-around;
-    width:105px;
-    height:25px;
-}
-.slider-dots img{
-    padding:0 8px;
-    float:left;
-    width:23px;
-    height: 23px;
-    
+
+.header-logo img {
+  width: 50px;
+  height: 50px;
+}
+.header-slider {
+  width: 1364px;
+  height: 524px;
+  border: 1px solid green;
+  margin-top: 187px;
+  margin-left: 38px;
+}
+.header-slider p {
+  font-size: 50px;
+  margin-top: 37px;
+  color: #fafafa;
+  margin-left: 91px;
+}
+.slider-buttons {
+  margin: 69px 96px;
+}
+.arrows {
+  position: absolute;
+  width: inherit;
+}
+.left-arrow {
+  margin: 120px -1px;
+  position: absolute;
+}
+.right-arrow {
+  margin-top: 120px;
+  position: absolute;
+  right: 1px;
+}
+.slider-headline {
+  font-size: 70px;
+  font-weight: 700;
+  font-style: normal;
+  margin-top: -17px;
+}
+.line {
+  margin: -16px 95px;
+}
+.slider-dots {
+  margin: 140px auto;
+  display: flex;
+  justify-content: space-around;
+  width: 105px;
+  height: 25px;
+}
+.slider-paragraph {
+  font-size: 50px;
+  width: 910px;
+}
+.focused {
+  width: 23px;
+  height: 23px;
+  border: 1px solid #fafafa;
+  background: #fafafa;
+}
+.unfocused {
+  width: 23px;
+  height: 23px;
+  border: 1px solid #fafafa;
+  background: none;
+}
+.explore-btn {
+  cursor: pointer;
+  color: #fafafa;
+  font-size: 16px;
+  text-transform: uppercase;
+  width: 150px;
+  height: 50px;
+  background: none;
+  border: 1px solid #fafafa;
+}
+.explore-btn:hover,
+.learn-btn:hover {
+  background: #d14b3a;
+  border: none;
+}
+.learn-btn {
+  cursor: pointer;
+  margin-left: 15px;
+  text-transform: uppercase;
+  width: 150px;
+  height: 50px;
+  color: #fafafa;
+  background: none;
+  border: 1px solid #fafafa;
 }
 .paralax-section {
-    background: url(/src/img/red-frame.png);
+  background: url(/src/img/red-frame.png);
   margin-top: -35px;
   width: $container-width;
   height: 700px;
+  z-index: 100;
   border: 1px solid black;
 }
-.paralax-headline{
-    @extend %headline;
+.paralax-headline {
+  @extend %headline;
 }
-.commercial-data {
-  margin-top: -40px;
-  width: $container-width;
-  height: 294px;
-  border: 1px solid black;
-  background: #656d7a;
+.paralax-quote {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  margin-top: -10vh;
+  color: #fafafa;
+}
+.quote-symbol {
+  color: #fafafa;
+  font-size: 150px;
+  opacity: 0.6;
+  margin-left: 15vw;
+  width: 51px;
+  height: 40px;
+}
+.quote {
+  font-style: italic;
+  font-size: 30px;
+  align-self: center;
+  width: 878px;
+  height: 71px;
+}
+.quote-author {
+  font-size: 30px;
+  margin-top: 5vh;
+  width: 878px;
```

Comment:
In general for this file:
- No need to use fixed sizes
- No need to use `vh` for indents everywhere (only where this really need)